plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -109,26 +112,32 @@ export default function transformProps(chartProps: ChartProps): EchartsTimeserie
             : 0,
       });
   });
+  let [yAxisMin, yAxisMax] = yAxisBounds || [];
+  if (yAxisMin === undefined && contributionMode === 'row' && stack) yAxisMin = 0;
+  if (yAxisMax === undefined && contributionMode === 'row' && stack) yAxisMax = 1;
   const echartOptions: echarts.EChartOption = {
     grid: {
+      ...defaultGrid,
       top: 30,
       bottom: zoomable ? 80 : 0,
       left: 20,
       right: 20,
-      containLabel: true,
     },
     xAxis: { type: 'time' },
     yAxis: {
+      ...defaultYAxis,
       type: logAxis ? 'log' : 'value',
-      min: contributionMode === 'row' && stack ? 0 : undefined,
-      max: contributionMode === 'row' && stack ? 1 : undefined,
+      // these might be NaN which break the y axis
+      min: yAxisMin || undefined,
```

Comment:
related to the comment above, but what else than int/undef could it be? Also I think `0` would be switched to undef here as `0 || undefined === undefined` which seems like it's not desirable if someone wanted `0` to be the bound

Reply:
Whoops!